plugins/exec/coqnum.ml

```diff
@@ -1,91 +0,0 @@
-(* FreeSpec
```

Comment:
Where do want to move this one?

Reply:
`plugins/exec/coqi63.ml` contains the necessary functions to translate `i63` terms (introduced by the `CoqFFI` theory; that are _signed_ integers for Coq implemented with native integers) into `int` OCaml value.